pycloudlib/ec2/cloud.py

```diff
@@ -216,7 +216,7 @@ def _find_image_serial(
             raise Exception("Could not find image: {}".format(image_id))
 
         image_name = images["Images"][0].get("Name", "")
-        serial_regex = r"ubuntu/.*/.*/.*-(?P<serial>\d+)$"
```

Comment:
I think we have an extra parenthesis here. After removing it, I can confirm this indeed fix the image serial for the provided image-id

Reply:
Fixed, thanks.